packages/documentation/src/templates/FormationReactTemplate.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
```

Comment:
This file is just `module-components.js` but renamed and moved to a new location. Also refactored to make it a function component.